### ISSUE TITLE
'Module not found' building with webpack 5 & @vue/cli 5 

### ISSUE THREAD

**Issue by nesnass** [bcakmakoglu/revue-draggable#223]
Including this plugin in an app built with Webpack 5 reports:
> Module not found: Error: Default condition should be last one

There is a rule in webpack 'enhanced-resolve' causing this error [here](https://github.com/webpack/enhanced-resolve/blob/02d99a2e6852adaf43fecfc3fed14d8d5c8df10b/lib/util/entrypoints.js#L455)

It can be easily fixed by moving "default" entry in package.json to be the last entry:
```
"exports": {
    ".": {
      "require": "./dist/revue-draggable.cjs.js",
      "default": "./dist/revue-draggable.es.js"
    },
```

**To Reproduce**
Build a Vue app using next Vue CLI version:
 - vue@3.2.29
 - @vue/cli 5.0.0-rc.2 
 - webpack@5.68
 - typescript@4.5.5

**Comment by bcakmakoglu** [comment]
Hello :) 
Thanks for reporting the issue, I will take a look at it and will get back to you asap 🙂

**Comment by bcakmakoglu** [comment]
The suggested fix is published in release 0.4.2 🥳
